synapse/storage/databases/main/events_worker.py

```diff
@@ -2589,6 +2602,42 @@ async def have_finished_sliding_sync_background_jobs(self) -> bool:
             )
         )
 
+    async def get_invite_count_by_user(self, user_id: str, from_ts: int) -> int:
+        """
+        Get the number of invites sent by the given user after the provided timestamp.
+
+        Args:
+            user_id: user ID to search against
+            from_ts: a timestamp in milliseconds from the unix epoch. Filters against
+            `events.received_ts`
```

Comment:
Subsequent wrapped lines should be indented to separate them from other args.

```suggestion
            from_ts: a timestamp in milliseconds from the unix epoch. Filters against
                `events.received_ts`
```